libraries/AP_Notify/NeoPixel.cpp

```diff
@@ -21,7 +21,7 @@
 #define AP_NOTIFY_NEOPIXEL_MAX_INSTANCES        4
 
 #ifndef HAL_NEOPIXEL_COUNT
```

Comment:
I think we should remove the #if block here and just use it to set the default for the parameter